wordpress-mobile/WordPress-iOS
# Unified About: Add Tracking

Initial PR for #17416.

Implements tracking for showing the about dialog, dismissing it, and pressing on any button within that screen.  It also covers the event tracking for "Legal and More".

## To test:

### Test 1:

- Go to Me > About
- Verify that you see:

🔵 Tracked: about_screen_shown <screen: main>

- Tap on any menu item and verify you see:

🔵 Tracked: about_screen_button_tapped <button: $button_name>

- Close the dialog, and verify you see:

🔵 Tracked: about_screen_dismissed <screen: main>

### Test 2: Test tapping buttons

Test tapping the buttons and make sure you get:

🔵 Tracked: about_screen_button_tapped <button: $button_name>

### Test 3: Legal and more

- Test tapping on "Legal And More"

🔵 Tracked: about_screen_shown <screen: legal_and_more>

- Tap on any menu item and verify you see:

🔵 Tracked: about_screen_button_tapped <button: $button_name>

- Dismiss the window by pressing back, tapping done or sliding it down.

🔵 Tracked: about_screen_dismissed <screen: legal_and_more>

## Regression Notes

1. Potential unintended areas of impact
2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
